package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "@ulaval/modul-vuex",
+    "version": "1.0.1-1",
+    "description": "MODUL Vuex utilities",
+    "author": "Martin Simard <martin.simard@dti.ulaval.ca>",
+    "private": false,
+    "scripts": {
+        "clean": "rimraf dist",
+        "reset": "rmdir /S /Q node_modules",
+        "build": "rimraf dist && tsc",
+        "tslint": "tslint -c conf/tslint.json -e 'src/**/*.d.ts' -s node_modules/custom-tslint-formatters/formatters -t grouped 'src/**/*.ts'",
+        "prepack": "npm run clean && npm run build",
+        "component": "node ./tests/meta/component-generator"
```

Comment:
What is this for ? I don't see the folder 'tests'.

Reply:
My bad, copy paste...